index.html

```diff
@@ -0,0 +1,234 @@
+            Listen to the speakers from various countries about the messages of
+            sharing and opening.
+          </p>
+        </article>
+        <article class="cards">
+          <img
+            class="cards__childs"
+            src="./img/program_icon_2.png"
+            alt="speaker"
+          />
+          <h3 class="main__h3 cards__childs">CC Exhibition</h3>
+          <p class="cards__childs">
+            Appreciate various creations applying CC license of artists,
+            organized from Art Center Nabi
+          </p>
+        </article>
+        <article class="cards">
+          <img
+            class="cards__childs"
+            src="./img/program_icon_3.png"
+            alt="speaker"
+          />
+          <h3 class="main__h3 cards__childs">Forum</h3>
+          <p class="cards__childs">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </article>
+        <article class="cards">
+          <img
+            class="cards__childs"
+            src="./img/program_icon_4.png"
+            alt="speaker"
+          />
+          <h3 class="main__h3 cards__childs">Workshop</h3>
+          <p class="cards__childs">
+            Try creating your own work using open source license rather than
+            just watching at it.
+          </p>
+        </article>
+        <article class="cards">
+          <img
+            class="cards__childs"
+            src="./img/program_icon_5.png"
+            alt="speaker"
+          />
+          <h3 class="main__h3 cards__childs">CC Ignite</h3>
+          <p class="cards__childs">
+            get opportunities to network with CC affiliates around the world,
+            also after the summit.
+          </p>
+        </article>
+      </div>
+      <button class="main__button">Join CC Global Summit 2015</button>
+
+      <p class="desktop-p">SEE THE WHOLE PROGRAM</p>
+    </main>
+    <section class="footer"></section>
+    <section class="footer2"></section>
+    <section class="desktop-main__partners">
+      <h2>Partner</h2>
+      <svg
+        class="desktop-main__section-svg"
+        xmlns="http://www.w3.org/2000/svg"
+        width="31"
+        height="3"
+        viewBox="0 0 31 3"
+        fill="none"
+      >
+        <line
+          x1="0.5"
+          y1="1.5"
+          x2="30.5"
+          y2="1.5"
+          stroke="#A04038"
+          stroke-width="2"
+        ></line>
+      </svg>
+      <div class="main__partners-logos">
+        <img src="./img/partners_airbnb 1.png" alt="airbnb logo" />
+        <img
+          src="./img/partners_daumkakaologo 1.png"
+          alt="daumkakaologo logo"
+        />
+        <img src="./img/partners_googlekorea.png" alt="google logo" />
+        <img src="./img/partners_mozilla.png" alt="mozilla logo" />
+        <img src="./img/partners_naver.png" alt="naver logo" />
+      </div>
+    </section>
+    <footer class="about_footer">
```

Comment:
- [optional] kindly reduce the logo size in the footer, so it can align with the original design
![Screenshot (885)](https://user-images.githubusercontent.com/65298048/226113875-2d43ee42-8c7c-45b8-8ef6-735363cddacc.png)